src/domain/event/queries/ticketmasterEventQuery.ts

```diff
@@ -0,0 +1,32 @@
+import { gql } from '@apollo/client';
+
+export const ticketmasterEventQuery = gql`
+  query ticketmasterEventQuery($eventId: ID!, $childId: ID!) {
+    event(id: $eventId) {
+      id
+      name
+      description
+      image
+      imageAltText
+      participantsPerInvite
+      occurrences: occurrences(upcoming: true, first: 1) {
```

Comment:
are they sorted properly?

Reply:
Should be, but that doesn't actually even really matter, because this is just a temporary workaround until we get just one URL for whole events.